build/verify-features.ts

```diff
@@ -20,15 +20,15 @@ for (let fileName of featuresDirContents) {
 		continue;
 	}
 
-	if (!importedFeatures.includes(featureMeta.id)) {
-		errors.push(`ERR: ${fileName} should be imported by \`/sources/refined-github.ts\`.`);
+	const featureId = fileName.replace('.tsx', '');
+	if (!importedFeatures.includes(featureId)) {
```

Comment:
I think it needs a `featureId as FeatureID`

Reply:
Yes that worked